BedrockCore.cpp

```diff
@@ -64,12 +64,8 @@ bool BedrockCore::peekCommand(BedrockCommand& command) {
                 response.content = newContent;
             }
         }
-    } catch (const char* e) {
```

Comment:
I'm a bit confused on this change and the associated one in processCommand.. Here we're calling "custom plugins", so it's all possible that any exception type is thrown, right?

Reply:
Yep, they can throw anything, though we only really supported `string` or `const char*`. Now, they need to throw `SException` instead of those types. There's a change in Auth so that auth does this.

the `catch (...)` line is actually fairly dangerous, which is why I removed it. The problem with it is that it can mask real, serious problems. What happens if something internally throws `std::bad_alloc`? If we catch `...`, then we just continue on as nothing happen ,and pretend it was a `500` from a command. I'd rather catch and fix these unknown sorts of errors.